### PR TITLE
[MAC] Enabling Mac Application Preference... menu and wiring it to open preferences file

### PR DESCRIPTION
**Pull request by nethip**
Enabling Application > Preferences... menu and wiring it to Debug > Open Preferences File

### CODE REVIEW

**Comment by nethip** [comment]
This PR might a bit scary w.r.t lot of changes in the `MainMenu.nib` files but I am completely trusting Mac's Interface Builder to have mapped each object to a menu. I did some level of unit testing, and the menus seem to be working fine.

**Comment by nethip** [comment]
cc @ryanstewart

File: appshell/cefclient_mac.mm [hunk 1]
```diff
@@ -199,6 +199,10 @@ - (IBAction)showAbout:(id)sender {
     g_handler->SendJSCommand(g_handler->GetBrowser(), HELP_ABOUT);
 }
 
+-(IBAction)handleOpenPreferences:(id)sender {
```
Comment (prafulVaishnav) [inline appshell/cefclient_mac.mm]: Please declare this function above

**Comment by prafulVaishnav** [comment]
In future, we could write unit test for it. LGTM merging it.

**Comment by nethip** [comment]
@prafulVaishnav  Updated this [waffle card](https://github.com/adobe/brackets/issues/11369) to add unit tests for this PR. Thanks!

**Pull request merged**